include/ensmallen_bits/nsga2/nsga2_impl.hpp

```diff
@@ -75,7 +75,7 @@ typename MatType::elem_type NSGA2::Optimize(
   if (lowerBound.n_rows == 1)
     lowerBound = lowerBound(0, 0) * arma::ones(iterate.n_rows, iterate.n_cols);
 
-  // Check if lower bound is a vector of a single dimension.
+  // Check if upper bound is a vector of a single dimension.
```

Comment:
Oops, thanks, great catch! :)

Reply:
@say4n, Hi, thanks. I'd love your reviews on the follow-up PR. Cheers! :)